yamllint/rules/quoted_strings.py

```diff
@@ -200,19 +229,29 @@ def check(conf, token, prev, next, nextnext, context):
     if (not token.plain) and (token.style == "|" or token.style == ">"):
         return
 
+    # Check value is quoted qoute
```

Comment:
Can you remove this comment? It's not very useful since the following code creates a var named `is_quoted_quote`.

Reply:
done